esx_circleprogressbar/__resource.lua

```diff
@@ -0,0 +1,16 @@
+resource_manifest_version '44febabe-d386-4d18-afbe-5e627f4af937'
```

Comment:
Why __resource and not fxmanifest? resource is really old and shouldn't be used

Reply:
I'm gonna update it!